empress/support_files/js/empress.js

```diff
@@ -3792,5 +3792,39 @@ define([
         this.redrawBarPlotsToMatchLayout();
     };
 
+    /**
+     * Returns the col variable value for the node.
+     *
+     * @param {Number} node Postorder position of a node in the tree.
+     * @param {String} col The name of the feature metadata column.
+     *
+     * @return {String} The col variable value for the node or undefined if
+     *                  no value exists.
+     */
+    Empress.prototype.getNodeFeatureMetadataValue = function (node, col) {
+        var colIndx = _.indexOf(this._featureMetadataColumns, col);
+        if (_.has(this._tipMetadata, node)) {
+            return this._tipMetadata[node][colIndx];
+        }
+        if (_.has(this._intMetadata, node)) {
+            return this._intMetadata[node][colIndx];
+        }
+        return undefined;
+    };
+
+    /**
+     * Checks to see if node has feature metadata.
+     *
+     * @param {Number} node Postorder position of a node in the tree.
+     *
+     * @return true if node has feature metadata else false.
+     */
+    Empress.prototype.hasFeatureMetadata = function (node) {
+        if (_.has(this._tipMetadata, node) || _.has(this._intMetadata, node)) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return _.has(this._tipMetadata, node) || _.has(this._intMetadata, node);
```

The current way this function is written is perfectly fine -- this suggestion just shortens it. Feel free to ignore this suggestion if you'd like :)

Reply:
Nice! :smile: